docs/dashboard.md

```diff
@@ -97,3 +97,70 @@ index=_internal source=*<addon_name>* ERROR
 ```
 
 > Note: <addon_name> is being replaced by the actual value during the build time.
+
+<br>
+# Custom components
+
+UCC also supports adding your own components to the dashboard. To do this, create a **dashboard_components.txt** file in the addon's base directory. 
```

Comment:
can we use `xml` format instead of `txt` to store custom dashboard?

Reply:
we can, in fact this was my first attempt but I changed it to .txt because if the file contains only row tags without any parent tag it is considered invalid. Since with this solution we only store individual components (rows) and not the entire, correct .xml file, I decided that this extension may be misleading. but if that's not the problem, I can go back to .xml.

Reply:
Let's pack it inside tag (like `custom-dashboard`), in case no such tag exist - throw an error an interrupt build.